Add files via upload

aikill opened the pull request:
Initial pandas implementation

aikill committed:
Add files via upload

Initial pandas implementation

mpooser commented:
Perhaps check to see if a csv file exists and if so, open it and save the data as a new row. Otherwise, create a csv, insert the data row, and export it as csv.

aikill committed:
Update acmdl.py

tried to comply with pycodestyle but the xpaths cannot be truncated without causing an error. the lines were shortened as much as apparently possible.
the new code to create/append the csv has been tested thoroughly and performs to expectations

mpooser reviewed (approved):
Seems to work fine on my end! Gonna go ahead and approve/merge.

mepcotterell commented:
@aikill @mpooser I'll approve the use of `# noqa` for long XPaths in future pull requests.
